src/js/utils/hooks.js

```diff
@@ -42,3 +42,41 @@ export const useDidUpdateEffect = (onUpdate, deps) => {
         firstRef.current = true;
     }, deps);
 };
+
+export function getUrlSearchParams(key) {
```

Comment:
* Is this used anywhere else than line 66 of this module? If not, no need to export. Further, I don't think a separate function is much benefit.
* The naming is not good. It doesn't get params, it gets a search term.